config/core/configmaps/autoscaler.yaml

```diff
@@ -172,3 +172,29 @@ data:
     # See the algorithm here: http://bit.ly/38XiCZ3.
     # TODO(vagababov): tune after actual benchmarking.
     activator-capacity: "100.0"
+
+    # initial-scale is the cluster-wide default value for the initial deployment
+    # replica size of a revision.
+    # This value cannot be set to 0 while allow-zero-initial-scale is false.
+    # If annotation "autoscaling.knative.dev/initialScale" is not set,
+    # or "autoscaling.knative.dev/initialScale" is set to 0 while
+    # allow-zero-initial-scale is false, the cluster wide initial-scale value will
+    # be used for the initial revision deployment size.
+    # If annotation "autoscaling.knative.dev/initialScale" is set to > 0,
```

Comment:
Reading through the rest of this configmap, I don't see anywhere else where we've talked about the overriding annotation in this much detail (or at all, in fact), even though pretty much every value in the config map can be overridden in the same way. Would we miss anything important if we just said:

```
initial-scale is the cluster-wide default value for the initial target scale of a revision
after creation, unless overridden by the "autoscaling.knative.dev/initialScale" annotation.
This value must be greater than 0 unless allow-zero-initial-scale is true.
```

(and leave the existing description of allow-zero-initial-scale to cover the edge case where that changes true->false).

Reply:
Yeah this is much simpler and I don't think we would be missing anything.